manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -110,7 +110,7 @@ spec:
     {{- end }}
   {{- else }}
     {{- if .Values.global.proxy.resources }}
```

Comment:
This should also be changed to proxy_init.

Reply:
Thanks! : )